pom.xml

```diff
@@ -154,68 +138,39 @@
         <scope>import</scope>
       </dependency>
       <dependency>
-        <groupId>com.fasterxml.jackson.core</groupId>
-        <artifactId>jackson-annotations</artifactId>
-        <version>${version.com.fasterxml.jackson}</version>
-      </dependency>
-      <dependency>
-        <groupId>com.fasterxml.jackson.core</groupId>
-        <artifactId>jackson-databind</artifactId>
-        <version>${version.com.fasterxml.jackson}</version>
-      </dependency>
-      <dependency>
-        <groupId>com.fasterxml.jackson.datatype</groupId>
-        <artifactId>jackson-datatype-jsr310</artifactId>
-        <version>${version.com.fasterxml.jackson}</version>
-      </dependency>
-      <!-- KIE -->
-      <dependency>
-        <groupId>org.optaplanner</groupId>
-        <artifactId>optaplanner-bom</artifactId>
+        <groupId>org.jboss.errai.bom</groupId>
+        <artifactId>errai-internal-bom</artifactId>
```

Comment:
is this really good to reference internal BOM instead of user facing BOM `org.jboss.errai.bom:errai-bom`

Reply:
Yes. It's not internal in the sense that it shouldn't be used by Errai users. `errai-internal-bom` only contains Errai artifacts whereas `errai-bom` adds 3rd party dependencies that Errai works with and it **includes** (imports) `errai-internal-bom`. In other words, by using `errai-bom` you get `errai-internal-bom` anyway and a bunch of other dependencies that we either don't use or manage ourselves.

Reply:
Also, all parent POMs in kiegroup import `errai-internal-bom`, and not `errai-bom` - that was probably the deciding factor for me to use it here too.

https://github.com/search?q=org%3Akiegroup+errai-internal-bom&type=Code

Reply:
ok, I originally recall the Errai only with errai-bom there was really only needed Errai dependencies for the usage.

That internal-errai-bom is useless I think we should ask to merge it into one BOM. 

Anyway I got the point here to import only managed errai deps without its build deps.